integration_tests/udev/disks.go

```diff
@@ -214,3 +214,21 @@ func (disk *Disk) Unmount() error {
 	}
 	return lastErr
 }
+
+func (disk *Disk) Resize(size int64) error {
+	units := [4]string{"", "K", "M", "G"}
```

Comment:
There should  be something in this package to help you with this.  https://pkg.go.dev/k8s.io/kubernetes@v1.22.0/pkg/api/v1/resource

Reply:
@RealHarshThakur I wasn't able to figure out how to use the package here. Could you please help out?

Reply:
Sourcegraph helps   : https://sourcegraph.com/search?q=context:global+resource.Quantity&patternType=literal .  
https://sourcegraph.com/github.com/kubernetes/kubernetes@64e422dc2ab5fab35026531194eefbfe66dc5aec/-/blob/staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go?L89:6

Reply:
Plan is to switch API fields to use this type too but we need to upgrade the CRD version.